build.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+#!/bin/env bash
```

Comment:
so did you actually change any logic here or just run it through some auto formatter? i dont see the point, afaik the script was working fine.

Reply:
Neither. Yes, `/bin/bash` will work fine on most systems, but in rare cases this path is not correct (NixOS comes to mind). `/bin/env bash` ensures it works across the board. It's just good practice.